pyseir/cli.py

```diff
@@ -63,7 +63,7 @@ def entry_point():
 
 @entry_point.command()
 def download_data():
-    cache_all_data()
```

Comment:
I would rather we clean up cache_all_data instead of replacing it with cache_county_case_data. We could just comment out the mobility and cache_public_implementations_data lines. I know they end up the same, but the cache_all_data function name is more informative, and in the future I'd like it to cache all relevant data.

Reply:
I know what you mean! I started implementing exactly what you described, but I decided that it might be a bit deceptive/confusing, since there would then be data sources in `load_data` that would not be considered `all_data`. 

If you strongly prefer just refactoring `cache_all_data` to only cache the county_case data, I will, though.